app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -236,6 +228,26 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun adaptToPrivateMode() {
```

Comment:
small nit: This method should probably be split up, or we should just move the click listener to live with the others :)

I would also like to consolidate how many places we're calling the `browsingModeManager` directly while it's still attached to the HomeActivity. An easy way to accomplish that would be

```
val isPrivate = (activity as HomeActivity).browsingModeManager.isPrivate
privateBrowsingButton.contentDescription = contentDescriptionForPrivateBrowsingButton(isPrivate = isPrivate)
```

```
fun contentDescriptionForPrivateBrowsingButton(isPrivate: Boolean): String {
    val resourceId = if (isPrivate) R.string.content_description_disable_private_browsing_button else R.string.content_description_private_browsing_button
    return getString(resourceId)
```